# HG changeset patch
# User Rik <rik@octave.org>
# Date 1375990552 25200
#      Thu Aug 08 12:35:52 2013 -0700
# Node ID 8b64a1fc3071521d790819049dcca73120ec7039
# Parent  ea5830dc6b19b67053f6a79fbc8ef8d7fd057ca7
test: Correct typo '%error' -> '%!error' to have test blocks run.

* scripts/general/quadgk.m: Correct typo '%error' -> '%!error'.

diff --git a/scripts/general/quadgk.m b/scripts/general/quadgk.m
--- a/scripts/general/quadgk.m
+++ b/scripts/general/quadgk.m
@@ -449,11 +449,11 @@ endfunction
 %!assert (quadgk (@(x) 1./sqrt (x),0,1), 2, 1e-6)
 %!assert (quadgk (@(x) abs (1 - x.^2),0,2, "Waypoints", 1), 2, 1e-6)
 %!assert (quadgk (@(x) 1./(sqrt (x) .* (x+1)),0,Inf), pi, 1e-6)
 %!assert (quadgk (@(z) log (z),1+1i,1+1i, "WayPoints", [1-1i, -1,-1i, -1+1i]), -pi * 1i, 1e-6)
 
 %!assert (quadgk (@(x) exp (-x .^ 2),-Inf,Inf), sqrt (pi), 1e-6)
 %!assert (quadgk (@(x) exp (-x .^ 2),-Inf,0), sqrt (pi)/2, 1e-6)
 
-%error (quadgk (@sin))
-%error (quadgk (@sin, -pi))
-%error (quadgk (@sin, -pi, pi, "DummyArg"))
+%!error (quadgk (@sin))
+%!error (quadgk (@sin, -pi))
+%!error (quadgk (@sin, -pi, pi, "DummyArg"))
